src/main/java/com/czertainly/api/model/core/certificate/X500RdnType.java

```diff
@@ -0,0 +1,79 @@
+package com.czertainly.api.model.core.certificate;
+
+import com.czertainly.api.model.common.enums.IPlatformEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Arrays;
+
+
+public enum X500RdnType implements IPlatformEnum {
```

Comment:
All enums names should follow convention of other enums - capital letters and words separated by `_`.